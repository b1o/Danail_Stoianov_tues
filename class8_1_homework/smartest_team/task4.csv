Ivan Petrov,4.79
Ivan Peichev,4.80
Stefan Petrov,5.73
Stefan Peichev,6.00
Peicho Peichev,5.79
Penka Petrova,2.10
Petroslava Ivanova,5.50
Ivaylo Grigorov,3.20
