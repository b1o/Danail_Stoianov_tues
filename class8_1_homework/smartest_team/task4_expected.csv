Stefan Peichev,50
Stefan Petrov,35
Peicho Peichev,35
Petroslava Ivanova,25
